EIPS/eip-666.md

```diff
@@ -0,0 +1,47 @@
+---
+eip: 666
+title: The Number of the Beast
+description: I beheld another beast coming up out of the earth; and he had two horns like a lamb, and he spake as a dragon.
+author: Beelzebub (@beelzebub)
+discussions-to: https://hell.com
+status: Draft
+type: Core
+---
+
+## Abstract
+
+The Address of the Beast is
+* 0x000000000000000000000000000000000000029A
+
+## Motivation
+
+We propose to assign an Ethereum address to The Beast. In the comming End Times ["... no man might buy or sell, save he that had the mark, or the name of the beast, or the number of his name."](https://www.kingjamesbibleonline.org/Revelation-Chapter-13/#17)  This implies that this address may be needed for the continued operation of the Ethereum in the End Times.
+
+## Specification
+
+[“Here is wisdom. Let him that hath understanding count the number of the beast: for it is the number of a man; and his number is Six hundred threescore and six.”](https://www.kingjamesbibleonline.org/Revelation-Chapter-13/#18)
+
+Decimal 666 is hexdecimal 0x29A.  This is the Number of the Beast.
+
+## Rationale
+
+In principle, nearly *2^256* numbers remain available, but for Biblical purposes the decimal number *666* was assigned in AD 95, which would be 0*x29A* in hex. That address and EIP number *666* remain available.
+
+## Backwards Compatibility
+
+Eternal.
+
+## Test Cases
+
+[Shut up the words, and seal the book, even to the time of the end.](https://www.kingjamesbibleonline.org/Daniel-Chapter-12/#4)
+
+## Reference Implementation
+
+[The Revelation of St John -- Albrecht Dürer](https://upload.wikimedia.org/wikipedia/commons/c/cf/Durer%2C_apocalisse%2C_03_ventiquattro_vecchi.jpg)
```

Comment:
External links should not be included.  A reference implementation can be included in `../assets/eip-4887/*`, or the section can be left out if a standalone reference implementation doesn't make sense.

Reply:
I'll move implementation to ../assets before this is merged, thanks.

Reply:
Even better -- how do I get an image to display?